.github/workflows/check-pull-request.yml

```diff
@@ -36,11 +28,13 @@ jobs:
         run: npm run format
       - name: Run linter
         run: npm run lint
-      - name: Start Postgres container
+      - name: Start database
         run: npm run docker:up:test
-      - name: Apply migrations to Postgres database
-        run: npm run prisma:migrate:test
+      - name: Wait for database to be ready
+        run: sleep 20
```

Comment:
Let's use sleep for now, but we can write a script for waiting mysql container to be active